typhon/objects/processes.py

```diff
@@ -57,17 +64,76 @@ def recv(self, atom, args):
 
 
 @autohelp
+class ProcessExitInformation(Object):
+    """
+    Holds a process' exitStatus and terminationSignal
+    """
+
```

Comment:
``_immutable_ = True`` would be good here. This object should be immutable, after all.